### ISSUE TITLE
Doesn't seem to be working in Sketch 71+

### ISSUE THREAD

**Issue by WideCircle** [abynim/sketch-selective-upload#14]
Hmmm - I just updated to Sketch 71.
After that Sketch uploads everything I have excluded to Sketch Cloud.

Best, Niklas

**Comment by TheManuz** [comment]
Can confirm, it's not working on Sketch 71.1

**Comment by abynim** [comment]
Sorry about this. Seems like a lot has changed in Sketch 71 around sharing/uploading documents to Sketch Cloud. This was kind of expected with the addition of the shared editing feature. 

Unfortunately I don't see any way to keep this plugin supported anymore since most of the new functionality seems to be written in Swift, which makes it inaccessible to the plugin interface. I'll check with the Sketch team to confirm if this change is permanent or if there's another way around it.

For now it would be best to request this feature in Sketch itself. I believe they have something similar on their list already but if we ask for it again they might bump up priority.

**Comment by RussellBishop** [comment]
Yep, I don't find this surprising at all. With Sketch for Teams, they want your documents to live _in_ Sketch Cloud, and for everyone to pull it from there. Uploading partial files therefore wouldn't make sense anymore, as it implies the Master is kept elsewhere.

Thanks for your work on this @abynim  it saved me a lot of time and effort for many months.

**Comment by TheManuz** [comment]
Me too, I've used your plugin for a long time, and even if Sketch offer an inspector with Sketch Cloud, I feel that I want to keep my files local, versioned inside my company Git.

Your plugin worked perfectly!
Thank you @abynim !
